src/components/compound/DlSlider/DlSlider.vue

```diff
@@ -216,7 +220,7 @@ export default defineComponent({
         }
 
         const handleResetButtonClick = () => {
-            onChange(initialValue.value)
+            onChange(props.defaultValue ?? min.value)
```

Comment:
1. Please notice to use `defaultValue` as the other props (populate it in the `toRefs(props)`) and use it with `.value` , without using `props.`
2. Keep the same behavior of `initialValue.value` instead of using the `min` value.